assets/src/edit-story/components/panels/size.js

```diff
@@ -18,101 +18,218 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
 /**
  * WordPress dependencies
  */
-import { useEffect, useState } from '@wordpress/element';
+import { useEffect, useState, useCallback } from '@wordpress/element';
 import { __, _x } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import { InputGroup } from '../form';
+import { Spacer, Button, Row, Numeric } from '../form';
 import { dataPixels } from '../../units';
+import { ReactComponent as Locked } from '../../icons/lock.svg';
+import { ReactComponent as Unlocked } from '../../icons/unlock.svg';
+import { ReactComponent as Fullbleed } from '../../icons/fullbleed.svg';
+import { ReactComponent as FlipVertical } from '../../icons/flip_vertical.svg';
+import { ReactComponent as FlipHorizontal } from '../../icons/flip_horizontal.svg';
+import Toggle from '../form/toggle';
 import { SimplePanel } from './panel';
 import getCommonValue from './utils/getCommonValue';
 
+const ExpandedNumeric = styled(Numeric)`
+  flex: 1;
+`;
+
+const BoxedNumeric = styled(ExpandedNumeric)`
+  padding: 6px 6px;
+  border: 1px solid ${({ theme }) => theme.colors.fg.v3};
+  border-radius: 4px;
+`;
+
+const FlipButton = styled(Button)`
+  border: none;
+  padding: 0;
+`;
+
+const FlipButtonIcon = styled.svg`
+  opacity: 0.54;
+`;
+
 function SizePanel({ selectedElements, onSetProperties }) {
```

Comment:
Since this is now a lot more than a size panel, it should be renamed and related properties too (like `PanelTypes.SIZE`)

Reply:
@barklund Just to clarify how to attack this, as far as I understand, this panel is grouping Sizing and Positioning controls, right? So, can you pls confirm if the list below are the expected changes? Thanks.

- Rename the file from `size.js` to `sizePosition.js`
- Rename `SizePanel` component name to `SizePositionPanel`
- Update references that are using `SIZE_POSITION` instead of `SIZE` (e.g. PanelTypes.SIZE_POSITION)

Reply:
Sounds good. And make sure all references to old panels are removed.